supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemJobCachingActorHelper.scala

```diff
@@ -1,45 +1,18 @@
 package cromwell.backend.sfs
 
-import akka.actor.{Actor, ActorRef}
+import akka.actor.Actor
 import com.typesafe.config.{Config, ConfigFactory}
-import cromwell.backend.BackendInitializationData
-import cromwell.backend.callcaching.JobCachingActorHelper
-import cromwell.backend.io.{JobPathsWithDocker, WorkflowPathsBackendInitializationData}
-import cromwell.backend.validation.{RuntimeAttributesValidation, ValidatedRuntimeAttributes}
+import cromwell.backend.standard.{StandardCachingActorHelper, StandardInitializationData}
 import cromwell.core.logging.JobLogging
 import cromwell.core.path.PathBuilder
 import net.ceedubs.ficus.Ficus._
 
-trait SharedFileSystemJobCachingActorHelper extends JobCachingActorHelper {
+trait SharedFileSystemJobCachingActorHelper extends StandardCachingActorHelper {
```

Comment:
Nope, there is no other SFS implementation besides Config, and even I get confused what goes where. In general I try to keep most of the static implementation in SFS, and anything that parses config code to turn it into WDL/scala goes in Config.

There used to be Local in addition to Config, but once I figured out a way to launch jobs async by ditching Scala's blocking `SimpleProcess` I was able to make Local a Config backend too.

Now that even more is in Standard, there's not much that is SFS specific that couldn't maybe go into Config.

Not cleaning up that mess in this PR though. Happy for one to file a ticket to sanitize the SFS/Config split later.